plugins/source/k8s/config/cloudquery-k8s-cluster-role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
if that's a test file it should either be removed or it should be used in some test